scipy/stats/_unuran/unuran_callback.h

```diff
@@ -129,6 +129,11 @@ double dpdf_thunk(double x, const struct unur_distr *distr)
     UNURAN_THUNK(PyFloat_FromDouble, "dpdf", 4);
 }
 
+double logpdf_thunk(double x, const struct unur_distr *distr)
+{
+    UNURAN_THUNK(PyFloat_FromDouble, "logpdf", 6);
```

Comment:
> Just so I eventually get familiar with this code, what is a "Thunk"

A [Thunk](https://en.wikipedia.org/wiki/Thunk) is just a wrapper around the Python function which converts all the inputs into Python objects, calls the Python function, and converts the returned objects back into C data types. This thunk is then called by UNU.RAN to evaluate the `"logpdf"` of the distribution.

> , and what is the number 6 for?

This is the length of the string `"logpdf"`. We can use `strlen` but since callbacks need to be fast, I avoided extra function calls since we know the length of `"logpdf"` at compile time.